hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMBlockProtocolServer.java

```diff
@@ -261,26 +264,33 @@ public List<DeleteBlockGroupResult> deleteKeyBlocks(
     List<DeleteBlockGroupResult> results = new ArrayList<>();
     Map<String, String> auditMap = Maps.newHashMap();
     ScmBlockLocationProtocolProtos.DeleteScmBlockResult.Result resultCode;
+    perfMetrics = scm.getPerfMetrics();
```

Comment:
Could you please initialize `SCMPerformanceMetrics` at a common place so it is accessible to all methods, instead of doing so under this specific method, allowing for the addition of metrics in the future?